### DISCUSSION TITLE
Vacation Mode (Pausing Reviews)

### DISCUSSION THREAD

**Discussion by C4MPM4N** [fabd/kanji-koohii#346] [1. Feedback & Feature Requests]
Hi,
First, I wanted to thank you for your great website and how much it has helped me so far. I was wondering if there is a way to pause the reviews, like some sort of vacation mode that websites like Bunpro have for their SRS system. In cases where someone can not do the reviews for a few days, they don't pile up.

**Comment by dongeraldo-create** [comment]
Agree! Especially as I am locked out because of a password problem and I probably have over 200 kanjis to catch up on! In a similar vein I would like to be able to smooth out the SRS so I do not get 2 one day and 12 the next...
